lib/python/gunittest/invoker.py

```diff
@@ -206,7 +206,8 @@ def _run_test_module(self, module, results_dir, gisdbase, location):
             shutil.rmtree(mapset_dir)
 
     def run_in_location(self, gisdbase, location, location_type,
-                        results_dir, test_list_in, test_list_out):
+                        results_dir, test_list_in='False',
+                        test_list_out='False'):
```

Comment:
Ok thank you. I initially had it set to `None`. In this case I would still need to test if it is `None` or `"False"` because this is the default value passed from the parser in `lib/python/gunittest/main.py`, passed from `lib/python/gunittest/multirunner.py`. There I can't use `None` because `subprocess.Popen() `would fail then. Do you think it would be better to pass the  `READ_MODULE_TEST_LIST` and `WRITE_MODULE_TEST_LIST` flags from `testsuite/examples/test_framework_GRASS_GIS_with_NC.sh` to the multirunner and then formulate two different calls to run `grass.gunittest.main`? Or should I test if it is `None` or `"False"` in `invoker.py`? Or better rename `"False"` in `multirunner.py` and `main.py` to a word that has no oher meaning in python? Thank you for your help

Reply:
Run [this](https://gist.github.com/pmav99/880d7e4de5d04098417fdecffbee9dea) with the following arguments:
```
python foo.py                                # no arguments -> i.e. you use the defaults
python foo.py --mypath ''                    # specify the empty string
python foo.py --mypath '/path/to/a/file'     # specify a proper path.
```

Hopefully, you will find it helpful. If not let me know.